crates/driver/src/domain/competition/solution/settlement.rs

```diff
@@ -248,6 +248,26 @@ impl Settlement {
             }
         };
 
+        {
+            // For testing purposes, always calculate CIP38 score, even if the score is not
+            // used.
+            let trades = self
+                .solutions
+                .iter()
+                .filter_map(|(id, solution)| match solution.scorable_trades() {
+                    Ok(trades) => Some(trades),
+                    Err(err) => {
+                        tracing::debug!(?id, ?err, "could not prepare scorable settlement");
```

Comment:
Can this happen without someone making a mistake? If not I think `warn`ing would be more appropriate.
```suggestion
                        tracing::warn!(?id, ?err, "could not prepare scorable settlement");
```

Reply:
obsolete code